### PR TITLE
Open Source Development - Bitbucket

### LINKED ISSUE #262: Add Resource - Open source development
I would like to add an useful resource which would help people get into open source development. 
please assign this issue to me under gssoc24?

### PR DESCRIPTION
**Pull request by PradnyaGaitonde**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

closes #262 

added resource for open-source development

https://bitbucket.org/

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x ] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [ x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

> fill the bracket with x. Something like this [x]

### CODE REVIEW

File: Open Source Development/readme.md [hunk 1]
```diff
@@ -109,6 +106,14 @@ Open source development involves collaborative creation of software where the so
...
     <td>An open source code editor developed by Microsoft, known for its versatility, extensibility, and support for various programming languages and technologies.</td>
     </tr>
+  <tr>
+    <td><a href="https://github.com/">GitHub</a></td>
+    <td>An essential platform for hosting and collaborating on open source projects, GitHub also offers extensive guides and resources for learning Git and contributing to projects.</td>
+  </tr>
+  <tr>
+    <td><a href="https://bitbucket.org/"></a>Bitbucket</td>
+    <td>What is BitBucket and How it works? An Overview and Its Use ...Bitbucket is a cloud-based Git-based service that allows developers to store, manage, and track code changes. It's designed for professional teams, providing a central place to collaborate on source code, plan projects, and test and deploy code.</td>
+  </tr>
```
Comment (jfmartinz) [inline Open Source Development/readme.md:116]:
```suggestion
  <tr>
    <td><a href="https://bitbucket.org/">Bitbucket</a></td>
    <td>What is BitBucket and How it works? An Overview and Its Use ...Bitbucket is a cloud-based Git-based service that allows developers to store, manage, and track code changes. It's designed for professional teams, providing a central place to collaborate on source code, plan projects, and test and deploy code.</td>
  </tr>
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
The Bitbucket link is not working because of this

File: Open Source Development/readme.md (hunk 1 above)
Comment (PradnyaGaitonde) [reply Open Source Development/readme.md]: Carried out the changes.

**Review by jfmartinz — APPROVED** [review]
Thanks

**Pull request merged**